internal-channels/README.md

```diff
@@ -0,0 +1,7 @@
+# internal-channels
+This directory contains non-version specific channels while the _public_ version specific channels stays in the [channels](../channels/) directory.
+These channels here are only for internal use (and they might become public as well in the future when more support for them is available).
+
+ART determines the releases in [candidate.yaml](candidate.yaml) and OTA promotes them to other channels channels in this folder with [stabilization-changes.py](hack/stabilization-changes.py).
```

Comment:
nit: you need either a repo-root-anchored `/hack/stabilization-changes.py` or a relative `../hack/stabilization-changes.py` as the link target here.

Reply:
Done.